integration-tests/src/test/resources/integration-test/openmrs-distro-spa-artifacts.properties

```diff
@@ -0,0 +1,13 @@
+name=Spa Artifact Example
+version=1.0.0-SNAPSHOT
+
+war.openmrs=2.6.9
+
+omod.spa=2.0.0
+
+spa.artifactId=openmrs-frontend-zl
+spa.groupId=org.pih.openmrs
+spa.version=1.3.0
+spa.include=openmrs-frontend-zl-1.3.0
```

Comment:
This is probably obvious and I'm just missing it, but why do we have to limit to "openmrs-frontend-zl-1.3.0"

Reply:
This is a hack/feature I had to add in, which I had originally hoped to avoid, because when we build our frontend zips we don't just include the files but we include them in a nested directory with this name.  So this is how I am providing an option to say "don't just unpack the entire zip as-is, but unpack it and copy in the contents of this subdirectory within the zip".